.devcontainer/bitwarden_common/docker-compose.yml

```
@@ -9,15 +9,16 @@ services:
     command: sleep infinity
 
   bitwarden_mssql:
-    image: mcr.microsoft.com/azure-sql-edge:latest
+    image: mcr.microsoft.com/mssql/server:latest
```

We should lock this to a major version to prevent accidental upgrades.
```suggestion
    image: mcr.microsoft.com/mssql/server:2022-latest
```